src/catkin_lint/main.py

```diff
@@ -103,7 +103,7 @@ def run_linter(args):
             pkgs_to_check += env.add_path(os.getcwd())
         else:
             sys.stderr.write("catkin_lint: no path given and no package.xml in current directory\n")
-            return 0
+            return 127
```

Comment:
Might be nice to use a constant from `os` here (one of the `os.EX_...` constants).

Reply:
Good idea, `EX_NOINPUT` seems to be the most appropriate:

```
 *	EX_NOINPUT -- An input file (not a system file) did not
 *		exist or was not readable.  This could also include
 *		errors like "No message" to a mailer (if it cared
 *		to catch it).
```

I don't know how portable this is

Reply:
It isn't too portable: the `os` module on Windows doesn't have these constants (but that's why they are in `os`).

I'm not sure how much of an issue this is: is `127` an exit code recognised to have any special significance on Windows?

Reply:
I have no idea I have never really programmed on Windows :grinning: